moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp

```diff
@@ -48,6 +49,11 @@ namespace trajectory_processing
 {
 const std::string LOGNAME = "trajectory_processing.time_optimal_trajectory_generation";
 constexpr double EPS = 0.000001;
+constexpr double DEFAULT_TIMESTEP = 1e-3;
+constexpr double DEFAULT_SCALING_FACTOR = 1.0;
+constexpr double DEFAULT_VELOCITY_LIMIT = 1.0;
```

Comment:
why? I thought we are good with [requiring users to set correct limits](https://github.com/ros-planning/moveit2/pull/1794)?  Scaling factor and timestep already have default arguments provided in the header. Let's not add more magic numbers to the code

Reply:
DEFAULT_SCALING_FACTOR and DEFAULT_TIMESTEP are gone now per your other comments.

DEFAULT_VELOCITY_LIMIT and DEFAULT_ACCEL_LIMIT were in the code previously, they simply weren't constants at the top of the file.  https://github.com/ros-planning/moveit/blob/8635f82e4823b970479f360216bbe8292f79e8e4/moveit_core/trajectory_processing/src/time_optimal_trajectory_generation.cpp#L927

I'll see how difficult this will be to remove in MoveIt1. IIRC it breaks some of the tests

Reply:
Do we want to break TOTG for the thousands of MoveIt users out there who don't have acceleration limits defined?

( ﾟｰﾟ)

Reply:
I thought we already had that discussion, but anyway it's not really in scope of this PR so I won't block it over this.